examples/parallel/parallel.cpp

```diff
@@ -70,6 +72,22 @@ struct client {
     std::vector<llama_token> tokens_prev;
 };
 
+static void printDateTime() {
+    std::time_t currentTime = std::time(nullptr);
+    std::cout << "\n\033[35mRUN PARAMETERS as at \033[0m" << std::ctime(&currentTime);
```

Comment:
We don't use `std::cout` in this project

Reply:
Changed to printf()

Reply:
All changes to `parallel.cpp` are on the new branch pushed a few moments ago. New branch is <origin/Update-load-parallel-prompt-file>. Longer external files produce longer processing times so I have tweaked llama.cpp LOG_TEE from %8.2f -> %10.2f as well at the end.